README.md

```diff
@@ -167,6 +212,22 @@ allowedTags: false,
 allowedAttributes: false
 ```
 
+#### "What if I want to allow empty attributes?"
+
+Very simple! Set `nonBooleanAttributes` to `[]`.
+
+```js
+nonBooleanAttributes: []
+```
+
+#### "What if I want to delete all empty attributes?"
+
+Also very simple! Set `nonBooleanAttributes` to `['*']`.
+
+```js
+nonBooleanAttributes: ['*']
+```
```

Comment:
Add a note that this will break common valid cases like `checked` and `selected` so for most ordinary HTML use cases it is best to avoid making this change.

Reply:
Updated